canvg.js

```diff
@@ -231,8 +231,14 @@
 			}
 			else if (window.DOMParser)
 			{
-				var parser = new DOMParser();
-				return parser.parseFromString(xml, 'text/xml');
+				try {
+					var parser = new DOMParser();

```

Comment:
@gabelerner Sorry I do not understand your comment. What is it your asking me to do?